test_unstructured/chunking/test_title.py

```diff
@@ -722,16 +725,210 @@ def it_provides_access_to_its_table(self):
 class Describe_TextSection:
     """Unit-test suite for `unstructured.chunking.title._TextSection objects."""
 
-    def it_provides_access_to_its_elements(self):
```

Comment:
`.elements` need no longer be public now, this property has been removed, which better encapsulates the section and would be dead code anyway.